packages/angular-workspace/.eslintrc.js

```diff
@@ -104,9 +106,27 @@ module.exports = {
             '@ni/eslint-config-angular/template'
         ],
         rules: {
-            // nimble-angular shouldn't provide user-visible strings so doesn't need i18n
-            // example-client-app is just for demo purposes and isn't anticipated to be localized
-            '@angular-eslint/template/i18n': 'off'
+            // Enable i18n template checking for the purpose of making sure to capture updates for the lint rules
+            // New entries should be added to the ignoreAttributeSets:
+            // See: https://github.com/ni/javascript-styleguide/blob/main/packages/eslint-config-angular/template/options.js
+            '@angular-eslint/template/i18n': [
```

Comment:
Should we add a sentence to nimble-angular CONTRIBUTING explaining the goal to add attributes in javascript-styleguide vs here? I'm also fine with this comment plus a mention at standup, your call.

Reply:
I think devs are more likely to discover it here when they hit the rule and for reviewers linking here in PRs would already be straightforward. Don't think CONTRIBUTING would give more context or be easier to link to so I think here is fine. I'll mention it at standup.